backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Metadaten.java

```diff
@@ -0,0 +1,170 @@
+package de.bund.digitalservice.ris.norms.domain.entity;
+
+import de.bund.digitalservice.ris.norms.utils.NodeCreator;
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import java.time.LocalDate;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+interface MetadataInterface {
```

Comment:
We now have a "Metadata" interface next to a "Metadaten" abstract. Language wise I was quite confused and thought that this must have slipped through. Letting that sink in, I now understand why. Actually no recommendation here.

Reply:
I could have put the interface in another file but it is just needed for the abstract class to be able to access the xpath of both types of enums within its scope. That is why it belongs somehow to the abstract class, which is specifying in the generic that the enum/class passed must implement that interface